docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -869,19 +869,10 @@ nginx.ingress.kubernetes.io/satisfy: "any"
 
 Enables a request to be mirrored to a mirror backend. Responses by mirror backends are ignored. This feature is useful, to see how requests will react in "test" backends.
 
-You can mirror a request to the `/mirror` path on your ingress, by applying the below:
+The mirror backend can be set by applying:
 
 ```yaml
-nginx.ingress.kubernetes.io/mirror-uri: "/mirror"
-```
-
-The mirror path can be defined as a separate ingress resource:
-
-```
-location = /mirror {
-    internal;
-    proxy_pass http://test_backend;
-}
+nginx.ingress.kubernetes.io/mirror-to-url: https://test.env.com/$request_uri
```

Comment:
I think `mirror-backend-url` might be a better annotation name.

Reply:
Why?

I thought in  `mirror-traffic-to`

Reply:
I'd suggested `mirror-target`, which will also be consistent with https://github.com/kubernetes/ingress-nginx/pull/5015/files#r375253721